dvc/remote/local/__init__.py

```diff
@@ -566,3 +591,69 @@ def unprotect(path_info):
     @staticmethod
     def protect(path_info):
         os.chmod(path_info.path, stat.S_IREAD | stat.S_IRGRP | stat.S_IROTH)
+
+    @classmethod
+    def _append_unpacked_suffix(cls, string):
+        return string + cls.UNPACKED_DIR_SUFFIX
+
+    def _get_unpacked_dir_info(self, checksum):
```

Comment:
How about 
```suggestion
    def _get_unpacked_dir_path_info(self, checksum):
```
so that we don't mixup `dir_info`(which is a list of relpath, checksum entries for dir) and `dir_path_info`